solutions/platinum/cses-2429.mdx

```diff
@@ -87,7 +87,7 @@ int main() {
 		return (fact[x] * inv[y] % MOD) * inv[x - y] % MOD;
 	};
 
-	// Function to calculate the number of ways to complete the grid
+	// Calculate the number of ways to complete the grid
 	auto f = [&](int i, int j, int k) {
```

Comment:
oh btw, this is partially preference, but I prefer to add a trailing return type for clarity